channeldb/invoices.go

```diff
@@ -262,7 +263,7 @@ type ContractTerm struct {
 	// PaymentPreimage is the preimage which is to be revealed in the
 	// occasion that an HTLC paying to the hash of this preimage is
 	// extended.
-	PaymentPreimage lntypes.Preimage
+	PaymentPreimage *lntypes.Preimage
```

Comment:
Update `godoc` to mention the significance of a `nil` value in this case?

Reply:
Added